app/api/roles.py

```diff
@@ -74,3 +80,45 @@ def before_delete_object(self, obj, kwargs):
                   'methods': {
                       'before_get_object': before_get_object
                   }}
+
+
+@role_misc_routes.route('/change-organiser', methods=['POST'])
+def change_organiser():
+    current_organiser_id = request.json.get('current_org_id', None)
+    new_organiser_id = request.json.get('new_org_id', None)
+    event_id = request.json.get('event_id', None)
```

Comment:
Please use proper marshmallow schema and handle it as other endpoints are handled. This route is not following JSON API spec